tests/test_sparkcontroller.py

```diff
@@ -41,6 +42,8 @@ def test_add_session():
     connection_string = "url=http://location:port;username=name;password=word"
     client = MagicMock()
     session = MagicMock()
+    session.kind = SESSION_KIND_SPARK
```

Comment:
Why did you add this? Your unit tests should be emitting events to memory and should not be using the real implementation.